packages/backend/src/clients/EmailClient/templates/imageNotification.html

```diff
@@ -1,490 +1,528 @@
-                                                                    <img
-                                                                        alt="linkedin"
-                                                                        class="social"
-                                                                        src="{{host}}/linkedin.png"
-                                                                /></a>
-                                                            </td>
-                                                        </tr>
-                                                        <tr>
-                                                            <td
-                                                                class="content-block powered-by"
-                                                            >
-                                                                <a
-                                                                    href="https://lightdash.com"
-                                                                    >lightdash.com</a
-                                                                >
-                                                            </td>
-                                                        </tr>
-                                                    </table>
-                                                </div>
-                                                <!-- END FOOTER -->
-                                            </td>
-                                        </tr>
-                                    </table>
-                                </td>
-                            </tr>
-                            <!-- END MAIN CONTENT AREA -->
-                        </table>
-                        <!-- END CENTERED WHITE CONTAINER -->
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>Simple Transactional Email</title>
+  <style>
+      /* -------------------------------------
+      GLOBAL RESETS
+  ------------------------------------- */
+
+      /*All the styling goes here*/
+
+      img {
+          border: none;
+          -ms-interpolation-mode: bicubic;
+          max-width: 100%;
+      }
+
+      body {
+          background-color: #e5e8eb;
+          font-family: 'Roboto', sans-serif;
+          -webkit-font-smoothing: antialiased;
+          font-size: 14px;
+          line-height: 1.4;
+          margin: 0;
+          padding: 0;
+          -ms-text-size-adjust: 100%;
+          -webkit-text-size-adjust: 100%;
+      }
+
+      table {
+          border-collapse: separate;
+          mso-table-lspace: 0pt;
+          mso-table-rspace: 0pt;
+          width: 100%;
+      }
+
+      table td {
+          font-family: 'Roboto', sans-serif;
```

Comment:
Are we set on Roboto ? @PriPatel

Reply:
@IrakliJani @yasmin-e Yeah, all our emails (sign up, forgot password etc) use Roboto. Not great, but consistent at least.